invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessLinks/LinksTab.js

```diff
@@ -69,10 +69,14 @@ export class LinksTab extends Component {
             />
           )}
         </Modal.Content>
-        <Modal.Actions>
-          <Button size="small" onClick={handleClose}>
-            {i18next.t("Close")}
-          </Button>
+        <Modal.Actions className="ui clearing segment">
+          <Button
+            size="small"
+            onClick={handleClose}
+            content={i18next.t("Cancel")}
```

Comment:
I changed the wording to "Cancel" instead of "Close" to follow the same nomenclature we have in other modals. 

They usually have a "Save" and "Cancel" buttons plus a "X" to close the modal.